.cache/v/cache/lastfailed

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This looks like something from pytest... We need to add .cache to .gitignore and not check it in.